bin/codecov.sh

```diff
@@ -43,22 +44,19 @@ mkdir -p "$COVERAGEDIR"
 if [[ -z ${MAXPROCS:-} ]];then
   MAXPROCS=$(($(getconf _NPROCESSORS_ONLN)/2))
 fi
-PIDS=()
-FAILED_TESTS=()
-
-declare -a PKGS
 
 function code_coverage() {
   local filename
   filename="$(echo "${1}" | tr '/' '-')"
-  ( go test \
+  go test \
     -coverpkg=istio.io/istio/... \
     -coverprofile="${COVERAGEDIR}/${filename}.cov" \
     -covermode=atomic "${1}" \
-    | tee "${COVERAGEDIR}/${filename}.report" ) &
-  local pid=$!
-  PKGS[${pid}]=${1}
-  PIDS+=("${pid}")
+    | tee "${COVERAGEDIR}/${filename}.report"  && RC=$? || RC=$?
+
+  if [[ ${RC} != 0 ]]; then
+    echo "${1}" | tee "${COVERAGEDIR}/${filename}.err"
```

Comment:
Should these be cleaned up? Otherwise if you rerun you'll see fails from previous runs.

Reply:
The output and err files are in the randomly generated $COVERAGEDIR in /tmp.